### PR TITLE
[Fix] Add serde attribute.

### PR DESCRIPTION
**Pull request by grao1991**
### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4250)

### CODE REVIEW

**Review by zekun000 — COMMENTED** [review]
why do we need this?

**Review by msmouse — APPROVED** [review]
Hmm.. I was under the impression we added this already.. shame..

**Comment by msmouse** [comment]
@zekun000  so that when we overwrite one of the options we don't need to list all the sibling configs at the same time.

File: config/src/config/storage_config.rs [hunk 1]
```diff
@@ -17,6 +17,7 @@ pub const TARGET_SNAPSHOT_SIZE: usize = 100_000;
 /// see <https://github.com/facebook/rocksdb/blob/master/include/rocksdb/options.h>
 /// for detailed explanations.
 #[derive(Copy, Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+#[serde(default, deny_unknown_fields)]
```
Comment (msmouse) [inline config/src/config/storage_config.rs:20]: Okay.. I assumed this was already set because I saw line 66...

**Review by zekun000 — APPROVED** [review]

**Pull request merged**